php/libraries/Module.class.inc

```diff
@@ -142,15 +165,32 @@ class Module extends \LORIS\Router\PrefixRouter implements RequestHandlerInterfa
     }
 
     /**
-     * Return the name of this module.
+     * Return the short name of this module. The short name is used
+     * in places such as the URL and by default matches the name of
+     * the directory that the module is housed in.
+     *
+     * It should not contain special characters.
      *
-     * @return string The module name
+     * @return string The machine friendly module name
      */
     public function getName() : string
     {
         return $this->name;
     }
 
+    /**
+     * Return the long name of this module. The long name is intended
+     * to be the friendly, human-readable name and can not be automatically
+     * derived. It may contain spaces or, capitalization. The long name
```

Comment:
```suggestion
     * derived. It may contain spaces or capitalization. The long name
```

Reply:
I don't think this is valid grammar.. there are only 2 things in the "or", not a comma separated list of things.

Reply:
Yeah I think we're agreeing? Your comment as the comma.

Reply:
oh, oops, I read the diff backwards.